devices/MS5611/CalibrationData.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Iot.Device.MS5611
+{
+    /// <summary>
+    /// MS5611 calibration data
+    /// </summary>
+    internal class CalibrationData
+    {
+        public int PressureSensitivity { private set; get; }
+        public int PressureOffset { private set; get; }
+        public int TemperatureCoefficientOfPressureSensitivity { private set; get; }
+        public int TemperatureCoefficientOfPressureOffset { private set; get; }
+        public int ReferenceTemperature { private set; get; }
+        public int TemperatureCoefficientOfTheTemperature { private set; get; }
+
+        /// <summary>
+        /// Reads data from device
+        /// </summary>
+        /// <param name="ms5611">Sensor object</param>
+        internal void ReadFromDevice(Ms5611 ms5611)
+        {
+            PressureSensitivity = ms5611.ReadRegister((int)CommandAddresses.MS5611_CMD_READ_PROM);
```

Comment:
Looking at this, few comments:
* most likelt the registers on +2, +4, etc have names as well or you can create one in the CommandAddress enum
* You should then adjust the ReadRegister to use this enum rather than int
* The cast will just happen in the ReadRegister function